### DISCUSSION TITLE
New Mind Map Node layouts: improving the presentation of the layouts

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#946] [General]
(I'm splitting from the thread #892, because it was getting hard to follow the multiple discussions there)

(this complements my comment https://github.com/freeplane/freeplane/discussions/892#discussioncomment-4687243)

Another idea is to present the icons in two groups of 3X3, like this

![image](https://user-images.githubusercontent.com/77707706/212560532-c3a8209e-666e-4c5c-9d0a-4c28cf46af16.png)

Distributing the icons like this may be more visually intuitive, because the user can easily understand the direction where the branch will spread to. For example, if he chooses the icon at the bottom left, the branch will grow to the direction of the bottom left of the map:
![image](https://user-images.githubusercontent.com/77707706/212560748-e76891f2-622e-4c6c-8d1b-51968aa3b3ed.png)

When resizing the sidebar: I suggest forcing the icons to be 3X3, instead of the dynamic changes that happen now, because the 3X3 layout is way easier to understand.

**Comment by dpolivaev** [comment]
I improved it in preview 1.11.1_08

**Reply by euu2021** [reply]
I think the first 3X3 group of icons is good. The second is confusing to me, though:
![image](https://user-images.githubusercontent.com/77707706/213882761-6a33cc52-0b7a-4244-8380-d4915fcaa906.png)

To me, it makes more sense the disposition that I suggested above, because it follows the directions of the branches:
![image](https://user-images.githubusercontent.com/77707706/213882864-1d269ae8-fe81-4199-b4ee-c620ec62979d.png)


To my eyes, it would be more clear if there was a divider and/or some spacing between each of those sections:
![image](https://user-images.githubusercontent.com/77707706/213882710-a7460ed1-b4b8-495c-b476-eaa9657f43cf.png)
